src/main/java/com/airlines/dao/impl/CrewMemberDaoImpl.java

```diff
@@ -13,8 +14,9 @@
 import java.sql.SQLException;
 
 public class CrewMemberDaoImpl implements CrewMemberDao {
-    private static final String INSERT_QUERY = "INSERT INTO crew_members " +
-            "(first_name, last_name, position, birthday, citizenship) VALUES (?, ?, ?, ?, ?);";
+    private static final String INSERT_QUERY = "INSERT INTO crew_members (first_name, last_name, position, birthday, citizenship) VALUES (?, ?, ?, ?, ?);";
+    private static final String SELECT_BY_ID_SQL = "SELECT * FROM crew_members WHERE id = ?;";
```

Comment:
SQL is a language.

SELECT_BY_ID_SQ -> SELECT_BY_ID_QUERY
UPDATE_CREW_MEMBERS_SQL -> UPDATE_QUERY

try to pay attention to consistancy and symmetry, you have three similar variables names, but used different approaches to naming:
_query
_sql
_sql

insert..
select by id ...
update crew member**S**...
in first ones , just method names, in last one - entity name.
And in plural form, I could try guessing, that you refers to crew_members as a table and you are updating this table with one row..
but it reads like you have List<CrewMember> and updates.

We rarely refer to table name in code. Usually, we refer to entity name - to prevent confusing, students - it's a List<Student> or reference to **students** table name